### PR TITLE
Update package.json with additional resolutions

### PR DESCRIPTION
**Pull request by leepavlouhmcts**
### Change description ###
added resolution

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by rayliangatos — APPROVED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@leepavlouhmcts - approved, following discussion we had on the need for `redis` and `mime` to be upgraded, too

**Review by Daniel-Lam — APPROVED** [review]
@leepavlouhmcts - these resolutions look OK to me... I suppose if I was being picky, some of them need a space after the `:` (i.e. `node-sass`, `tree-kill`, and `acorn`) - but no major issue

**Review by Daniel-Lam — DISMISSED** [review]
@leepavlouhmcts - looks good; approved

File: test/e2e/support/reports.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
```
Comment (Daniel-Lam) [inline test/e2e/support/reports.js:1]: Probably better to remove these commented-out lines if no longer required.

File: test/e2e/support/reports.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+// const fs = require('fs');
+// const mkdirp = require('mkdirp');
 
-const xmlReports = `${process.cwd()}/reports/xml`;
+// const xmlReports = `${process.cwd()}/reports/xml`;
```
Comment (Daniel-Lam) [inline test/e2e/support/reports.js:5]: Same for this commented-out line.

File: test/e2e/support/reports.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-// var createXmlDir = (function() {
-//     if (!fs.existsSync(xmlReports)) {
-//         mkdirp.sync(xmlReports);
-//     }
-// })();
+// // var createXmlDir = (function() {
```
Comment (Daniel-Lam) [inline test/e2e/support/reports.js:7]: And these commented-out lines.

File: test/e2e/support/reports.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+// //     if (!fs.existsSync(xmlReports)) {
+// //         mkdirp.sync(xmlReports);
+// //     }
+// // })();
 
-reporter.config({ targetDir: xmlReports });
+// reporter.config({ targetDir: xmlReports });
```
Comment (Daniel-Lam) [inline test/e2e/support/reports.js:13]: And this one.

File: test/e2e/support/reports.js [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 
-module.exports = reporter;
+// module.exports = reporter;
```
Comment (Daniel-Lam) [inline test/e2e/support/reports.js:15]: And this one, too.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@leepavlouhmcts - review completed; just a few commented-out lines that you can probably remove. The rest looks OK.

**Review by paul-graham — APPROVED** [review]
A little concerned that we've got some libraries going up by major versions. I think this will need a full regression test. However, they all look like reasonable changes.

**Review by Daniel-Lam — APPROVED** [review]

File: test/e2e/support/reports.js (hunk 1 above)
Comment (leepavlouhmcts) [reply test/e2e/support/reports.js:1]: i have removed the contents of the entire file

File: test/e2e/support/reports.js (hunk 2 above)
Comment (leepavlouhmcts) [reply test/e2e/support/reports.js:5]: i have removed the contents of the entire file

File: test/e2e/support/reports.js (hunk 3 above)
Comment (leepavlouhmcts) [reply test/e2e/support/reports.js:7]: i have removed the contents of the entire file

File: test/e2e/support/reports.js (hunk 4 above)
Comment (leepavlouhmcts) [reply test/e2e/support/reports.js:13]: i have removed the contents of the entire file

File: test/e2e/support/reports.js (hunk 5 above)
Comment (leepavlouhmcts) [reply test/e2e/support/reports.js:15]: i have removed the contents of the entire file

**Pull request merged**